modules/storage.js

```diff
@@ -0,0 +1,31 @@
+export default class Storage {
+  constructor () {
+    this.Books = []
+  }
+
+  static addBook (newBook) {
+    this.Books.push(newBook)
+    localStorage.setItem('Books', JSON.stringify(this.Books))
+  }
+
+  static removeFromBooks (target) {
+    const removeBook = target.previousElementSibling.firstElementChild.textContent
+
+    this.Books.filter((book, index) => {
+      if (book.title === removeBook) {
+        this.Books.splice(index, 1)
+      }
+      return this.Books
+    })
+    localStorage.setItem('Books', JSON.stringify(this.Books))
+  }
+
+  static getBooksFromStorage () {
+    if (localStorage.getItem('Books') === null) {
+      this.Books = []
+    } else {
+      this.Books = JSON.parse(localStorage.getItem('Books'))
+    }
+    return this.Books
+  }
```

Comment:
- [ ] Please it would be better if make these methods as `Arrow functions` instead of `normal functions` that will make use the `Es6` features

Reply:
Hi @youmari I implemented the changes, but the syntax is causing a linter error as you can see, could you give me some insight as to why? Thanks.

Reply:
@Amtresu as the requirement of this project is to use arrow function instead of function keyword in normal function but according to Mozilla documentation it is not recommended to use the static arrow function as methods 
so what you should make the changes back to normal function and send for another review 
- Sorry for the delay